test/test_gc.c

```diff
@@ -7,6 +7,10 @@
 
 #define UNUSED(x) (void)(x)
 
+#if defined(_MSC_VER)
+#define __builtin_frame_address(x)  ((void)(x), _AddressOfReturnAddress())
+#endif
+
```

Comment:
Since `gc_test.c` `#include`s the implementation from `gc.c` you should be able to drop these lines (we need to include the `.c` file to gain access to the `static` functions.

Reply:
Hi @mkirchner, yeah my bad, I fixed it, thanks.